src/main/java/racing/core/domain/Cars.java

```diff
@@ -0,0 +1,43 @@
+package racing.core.domain;
+
+import racing.core.dto.TrackInfo;
+import racing.core.dto.Trial;
+import racing.core.patterns.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private static final int WINNER = 0;
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        if (cars.isEmpty()) {
+            throw new IllegalArgumentException("경주에 참가할 자동차가 존재하지 않습니다.");
+        }
+        this.cars = cars;
+    }
+
+    public Trial nextTrial(MoveStrategy movement) {
+        List<TrackInfo> tracks = new ArrayList<>();
+        cars.stream().forEach(car -> tracks.add(car.move(movement)));
```

Comment:
이렇게 줄일 수도 있습니다.
```suggestion
        cars.forEach(car -> tracks.add(car.move(movement)));
```
아니면 이렇게 사용도 가능합니다:)
```
List<TrackInfo> tracks = cars.stream()
       .map(car -> car.move(movement))
       .collect(Collectors.toList());
```

Reply:
추천해주신 방법이 스트림을 더 자연스럽게(?) 사용한 것 같아요! 참고하겠습니다!